General/Operations/mission.py

```
@@ -15,7 +34,11 @@ def upload_payload_drop_mission(vehicle_connection, payload_object_coord):
     # - The function will block until all waypoints are uploaded or a failure occurs.
 
     try:
-        count = 3
+        data = read_mission_json()
+        waypoints = data.get("waypoints", {})
+        entry = waypoints.get("entry")
+        exit = waypoints.get("exit")
```

Can we wrap this in checks at the end to make sure entry and exit != null? I don't want this to silently fail